README.md

```diff
@@ -1,32 +1,32 @@
-# taurus
-Python binding for Citrine's nextgen data model (codename: taurus). 
+# GEMD-python 
+Python binding for Citrine's nextgen data model, GEMD. 
 
 This package provides a framework for storing information about the processes that create materials, the materials themselves, and measurements performed on those materials. 
 
 ## Usage
 
-To install `taurus`, you can simply:
+To install `gemd`, you can simply:
 ```
-$ pip install taurus-citrine
+$ pip install gemd
 ```
 
 Detailed documentation of the `GEMD` data model can be found in the [language-agnostic documentation](https://citrineinformatics.github.io/gemd-docs/).
-Documentation of this package can be found [here](https://citrineinformatics.github.io/taurus/).
+Documentation of this package can be found [here](https://citrineinformatics.github.io/gemd/).
```

Comment:
Are we migrating the location of this repo as well, or just renaming things?

Reply:
We will be, yes.